### PR TITLE
Mobile layout

### LINKED ISSUE #752: Rooms page is too wide on mobile device
When viewed on mobile device (Android 5.1, latest Chrome version, FHD resolution) Rooms page is too wide - so there is horizontal scroll and icon-bar is not visible unless you scroll to the right. This happens only on Rooms page - other pages are rendered just fine (without horizontal scroll).

### PR DESCRIPTION
**Pull request by alexagr**
layout fixes on mobile devices
resolve #752 and #753

### CODE REVIEW

**Comment by alexagr** [comment]
Sorry for too many commits - I fixed alignment of some existing block and got hound for errors that were in it before I event touched it; hopefully they are fixed now.

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -664,20 +673,25 @@ a {
...
- @media (max-width: 990px) {
-    .title-inline {
-      width: 50%;
-      float: left;
-    }
-    .track-names {
-      margin-top :0;
-    }
+.track-names {
+  color: #000000;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: Please use config.scss to use variable color name instead of hex color value

Comment (alexagr) [inline src/backend/_scss/application.scss]: sorry - misinterpreted the hound report - fixed in the latest commit

Comment (alexagr) [inline src/backend/_scss/application.scss]: right - I misinterpreted the hound report - fixed in the latest commit

**Comment by alexagr** [comment]
Hugh - for some reason my line comment disappears - already wrote it twice... Anyway - the problem with "black" color is fixed.

File: src/backend/_scss/application.scss [hunk 2]
```diff
@@ -664,21 +673,26 @@ a {
...
- @media (max-width: 990px) {
-    .title-inline {
-      width: 50%;
-      float: left;
-    }
-    .track-names {
-      margin-top :0;
-    }
+.track-names {
+  color: black;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @alexagr it should not be 'black', it should be $black that is a sass variable in config.scss.

**Comment by aayusharora** [comment]
@alexagr Please see the above comment.

**Comment by alexagr** [comment]
oh - that makes sense - fixed

**Comment by aayusharora** [comment]
Looking Nice @alexagr Merging this.

**Pull request merged**